src/dataclay/dataclay_object.py

```diff
@@ -466,7 +470,10 @@ def get_by_id(cls, object_id: UUID) -> DataClayObject:
     @classmethod
     @tracer.start_as_current_span("get_by_alias")
     async def _get_by_alias(cls: Type[T], alias: str, dataset_name: str = None) -> T:
-        return await get_runtime().get_object_by_alias(alias, dataset_name)
+        try:
+            return await get_runtime().get_object_by_alias(alias, dataset_name)
+        except:
```

Comment:
A naked except clause incomodates me. It would be better to be more specific (otherwise, a connectivity error due to gRPC may end up raising an AliasDoesNotExistError, which is not ideal). I don't know if that's feasible.